docker/web.Dockerfile

```diff
@@ -3,18 +3,20 @@ FROM golang:alpine
 # Required because go requires gcc to build
 RUN apk add build-base
 
+RUN apk add --no-cache git
```

Comment:
@paudelgaurav Why was `--no-cache` added?

Reply:
I added --no-cache so that if git was updated on Alpine Linux, we will rebuild the image. I would love to hear your insights.